src/schemas/21-526EZ/schema.js

```diff
@@ -143,7 +143,8 @@ let schema = {
     }),
     fullName: fullNameDef,
     phone: definitions.usaPhone,
-    form4142: definitions.form4142
+    form4142: definitions.form4142,
+    centralMailAddress: definitions.centralMailAddress
```

Comment:
:thinking: I see why this is required, but for posterity, can you add a comment saying this is for the 4142 schema since it's not immediately obvious?

Reply:
Added comment for 4142 schema